src/profile/profile.view.ts

```diff
@@ -1,46 +1,30 @@
-/*   Display A Public Profile Pane
- **
- ** This is the subject's primary representation in the world.
- ** When anyone scans the QR code of their webid on theor card, it takes gthem
- ** to here and here alone.  Thiks has better be good.  This has better be
- ** worth the subjectjoing solid for
- ** - informative
- **
- ** Usage: paneRegistry.register('profile/profilePane')
- ** or standalone script adding onto existing mashlib.
+/**
+ * Display A Public Profile Pane
+ *
+ * This is the subject's primary representation in the world.
+ * When anyone scans the QR code of their WebID on their card, it takes them
+ * to here and here alone.  This has better be good.  This has better be
```

Comment:
```suggestion
 * to here and here alone.  This had better be good.  This had better be
```
Sorry I missed these before!